neuralmonkey/runners/rnn_runner.py

```diff
@@ -281,6 +288,11 @@ def collect_results(self, results: List[Dict]) -> None:
         if self._time_step == self._decoder.max_output:
             top_batch = self._to_exapand[-1].decoded.T
             decoded_tokens = self._vocabulary.vectors_to_sentences(top_batch)
+
+            if self._postprocess is not None:
```

Comment:
Mně přijde, že jsem to dostatečně zabezpečil, že je tam ta typová anotace (kdyby u toho předchozího bugu byla, tak by mi bylo mypy vynadalo). Navíc, až tomáš udělá kontrolu typů při budování konfigurace, tak to bude stopro bezpečný.

Navíc `is not None` je nejvíc pythonic:
https://www.python.org/dev/peps/pep-0008/#id49